.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - "node"
 install:
-  - npm install
+  - yarn --ignore-optional
```

Comment:
@jianliao `yarn --ignore-optional` seems to be required to run outside of the Adobe network due to our optional dependencies... Also wondering why `@adobe/spectrum-css-workflow-icons` has optional dependencies on `@a4u` packages? I don't understand why they're there.

Reply:
I checked in all the workflow icon SVG files to [spectrum-css-workflow-icons repo](https://github.com/adobe/spectrum-css-workflow-icons), but the source of these icons are still coming from A4U. When A4U updates those packages, I can easily get the update by running `npm install` within adobe corp network. That is the reason I add them as optional dependencies.

For external contributor, they can't access A4U artifactory. If they want to modify the existing workflow icon, they still can fully setup the project with `npm install`. Those optional dependencies would not block them. WDYT?